R/test-directory.R

```diff
@@ -91,11 +86,6 @@ test_dir <- function(path,
   source_test_setup(path, env)
   on.exit(source_test_teardown(path, env), add = TRUE)
 
-  # Promote retirement stages except on CRAN
-  if (identical(Sys.getenv("NOT_CRAN"), "true")) {
-    withr::local_options(list(lifecycle_verbose_retirement = TRUE))
```

Comment:
This doesn't appear to do anything — there's no mention of "lifecycle_verbose_retirement" in the lifecycle package.